dsp/main.js

```diff
@@ -31,7 +31,7 @@ function shouldRender(prevState, nextState) {
 // on the result of our `shouldRender` check.
 globalThis.__receiveStateChange__ = (serializedState) => {
   const state = JSON.parse(serializedState);
-
+  console.log(state);
```

Comment:
Ok if we delete this?

Reply:
Yes, this was just a quick way to get an immediate stream of log statements when testing.